swarm/tools/validation/cli.py

```
@@ -0,0 +1,169 @@
+    )
+
+    args = parser.parse_args()
+
+    # Parse registry first (needed for both validation and JSON output)
+    try:
+        registry = parse_agents_registry()
+    except SystemExit:
+        if args.json:
+            error_output: Dict[str, Any] = {
+                "version": "1.0.0",
+                "timestamp": datetime.now(timezone.utc).isoformat(timespec="seconds"),
```

Ruff (0.14.13)

Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)